### PR TITLE
docs: add sentinel command example for RHEL family

### PR DESCRIPTION
**Pull request by khuedoan**
Add example for RHEL.

Fixes https://github.com/weaveworks/kured/issues/498

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,9 +128,22 @@ values with `--reboot-sentinel` and `--period`. Each replica of the
...
+that a reboot is required.
+
+For example, if you're using RHEL or its derivatives, you can
+set the sentinel command to `sh -c "! needs-restarting --reboothint"`
+(by default the command will return `1` if a reboot is required,
+so we wrap it in `sh -c` and add `!` to negate the return value).
+
+```yaml
+configuration:
+  rebootSentinelCommand: sh -c "! needs-restarting --reboothint"
```
[the remark this answers is not in the document]
Comment (khuedoan) [reply README.md:145]: Yes, Helm still creates the same output YAML with or without quotes:

```yaml
# ...
          command:
            - /usr/bin/kured
          args:
            - --ds-name=release-name-kured
            - --ds-namespace=default
            - --reboot-sentinel-command=sh -c "! needs-restarting --reboothint"
            - --reboot-command=/bin/systemctl reboot
            - --log-format=text
# ...
```

I've tested both [in my homelab](https://github.com/khuedoan/homelab/commit/20731cdcda45ae7bd6607a1b59435aa6ead2ce52)

**Review by jackfrancis — APPROVED** [review]
/lgtm

**Comment by chadswen** [comment]
Looks great. This will save people some trial and error time.

Not to derail, but has anyone else tried something like `--reboot-sentinel-command=sh -c "[ -f /var/run/reboot-required ] || ( [ -x /usr/bin/needs-restarting ] && ! needs-restarting -r )"` to support clusters with a mix of both Red Hat and Debian family nodes? Would that worth including here for better compatibility?

**Comment by khuedoan** [comment]
I don't have a cluster with mixed distros so I can't test that, therefore it won't be included in this PR, but in theory it _should_ work.

**Comment by portno12** [comment]
while i single quoted the string for safety the command worked perfectly for me on fedora 35

**Review by ckotzbauer — APPROVED** [review]

**Pull request merged**